plugins/auth-backend/src/lib/oauth/OAuthAdapter.ts

```diff
@@ -98,7 +103,8 @@ export class OAuthAdapter implements AuthProviderRouteHandlers {
     const scope = req.query.scope?.toString() ?? '';
     const env = req.query.env?.toString();
     const origin = req.query.origin?.toString();
-
+    const redirectUrl = req.query.redirectUrl?.toString();
+    const authFlow = req.query.authFlow?.toString();
```

Comment:
Since we're already well within an auth context I think we can just call this `flow`